dwave/system/composites/linear_ancilla.py

```diff
@@ -0,0 +1,219 @@
+# Copyright 2024 D-Wave Inc.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+
+"""Composite to implement linear coefficients with ancilla qubits biased with flux-bias offsets.
+"""
+
+import numbers
+
+from collections import defaultdict
+from typing import Sequence, Mapping, Any
+
+import dimod
+import numpy as np
+
+from dimod.decorators import nonblocking_sample_method
+
+
+__all__ = ["LinearAncillaComposite"]
+
+
+class LinearAncillaComposite(dimod.ComposedSampler, dimod.Structured):
+    """Implements linear biases as ancilla qubits polarized with strong flux biases.
+
+    Linear bias :math:`h_i` of qubit :math:`i` is implemented through a coupling 
+    :math:`J_{ij}` between the qubit and a neighboring qubit :math:`j` that has a 
+    large flux-bias offset.
+
+    Args:
+    child_sampler (:class:`dimod.Sampler`):
+        A dimod sampler, such as a :class:`~dwave.system.samplers.DWaveSampler()`, 
+        that has flux bias controls.
```

Comment:
Please add a small code example, ideally using the EmbeddingComposite or such. See [example in TilingComposite](https://docs.ocean.dwavesys.com/en/stable/docs_system/reference/composites.html#tilingcomposite) for the format and preferred size.